frontend/package.json

```diff
@@ -1,43 +1,44 @@
 {
-  "name": "frontend",
-  "private": true,
-  "license": "MIT",
-  "version": "0.0.0",
-  "type": "module",
-  "scripts": {
-    "dev": "vite",
-    "build": "wasm-pack build catlog-wasm && tsc -b && vite build",
-    "preview": "vite preview"
-  },
-  "dependencies": {
-    "@automerge/automerge": "^2.2.4",
-    "@automerge/automerge-repo": "^1.2.0",
-    "@automerge/automerge-repo-network-websocket": "^1.2.0",
-    "@automerge/automerge-repo-storage-indexeddb": "^1.2.0",
-    "@automerge/prosemirror": "^0.0.12",
-    "@corvu/popover": "^0.2.0",
-    "@corvu/resizable": "^0.2.2",
-    "@solid-primitives/active-element": "^2.0.20",
-    "@solid-primitives/context": "^0.2.3",
-    "@solid-primitives/keyboard": "^1.2.8",
-    "@solid-primitives/pointer": "^0.2.19",
-    "@viz-js/viz": "^3.7.0",
-    "catlog-wasm": "file:catlog-wasm/pkg",
-    "prosemirror-commands": "^1.5.2",
-    "prosemirror-keymap": "^1.2.2",
-    "prosemirror-model": "^1.21.3",
-    "prosemirror-state": "^1.4.3",
-    "prosemirror-view": "^1.33.8",
-    "solid-js": "^1.8.17",
-    "uuidv7": "^1.0.1"
-  },
-  "devDependencies": {
-    "typedoc": "^0.26.5",
-    "typescript": "^5.2.2",
-    "vite": "^5.3.1",
-    "vite-plugin-solid": "^2.10.2",
-    "vite-plugin-top-level-await": "^1.4.1",
-    "vite-plugin-wasm": "^3.3.0",
-    "wasm-pack": "^0.13.0"
-  }
+    "name": "frontend",
+    "private": true,
+    "license": "MIT",
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
```

Comment:
We should have a `format` script so that we don't have to remember how to run the formatter.

Reply:
Sounds good, I'll also add a `lint` script